bigchaindb/tendermint/lib.py

```diff
@@ -271,19 +259,15 @@ def get_transaction(self, transaction_id, include_status=False):
             transaction = Transaction.from_dict(transaction)
 
         if include_status:
-            return transaction, self.TX_VALID if transaction else None
+            return transaction, 'valid' if transaction else None
```

Comment:
I think the `'valid'` should go away here too.

Reply:
Then I need to change the whole function and remove `include_status` too. I can do it if we want that.

Reply:
I think `include_status` is only used in the above `if` condition?

Reply:
yes, it's only in the if.